src/ResourceManager/ManagementPartner/ChangeLog.md

```diff
@@ -19,6 +19,8 @@
 -->
 ## Current Release
 
-## Version 0.1.0
-* First release of Management Partners cmdlets
+## Version 0.1.1
```

Comment:
@jeffrey-ACE one small comment -- you don't need to add a new header, you can put the text you're adding under the `## Current Release` header